apps/src/studio/levels.js

```diff
@@ -54,6 +54,64 @@ function saySpriteRequiredBlock(options) {
 }
 
 /**
+ * Constructs a required block definition to match "move [sprite] [dir]" blocks
+ * @param options (all optional):
```

Comment:
Is this the standard documentation format?  I've used something like this that I found elsewhere in our code..

 * @returns {Object} outcome
 * @returns {boolean} outcome.exists Whether the level even has any of these criteria.
 * @returns {boolean} outcome.achieved false if any of the criteria was required but not met.
 * @returns {string} outcome.message A custom message for the first of the failing criteria.